bin/verdicts.py

```diff
@@ -0,0 +1,452 @@
+from pathlib import Path
+import shutil
+import sys
+from enum import Enum
+
+from util import ProgressBar
+import config
+from colorama import Fore, Style
+
+
+class Verdict(Enum):
+    """The verdict of a testcase or testgroup"""
+
+    ACCEPTED = 1
+    WRONG_ANSWER = 2
+    TIME_LIMIT_EXCEEDED = 3
+    RUNTIME_ERROR = 4
+    VALIDATOR_CRASH = 5
+    COMPILER_ERROR = 5
+
+    def __str__(self):
+        return {
+            Verdict.ACCEPTED: 'ACCEPTED',
+            Verdict.WRONG_ANSWER: 'WRONG ANSWER',
+            Verdict.TIME_LIMIT_EXCEEDED: 'TIME LIMIT EXCEEDED',
+            Verdict.RUNTIME_ERROR: 'RUNTIME ERROR',
+            Verdict.COMPILER_ERROR: 'COMPILER ERROR',
+        }[self]
+
+
+def to_string(v: Verdict | None):
```

Comment:
wouldnt it be simpler to add something like `get_color` and then implement this as f`{get_colot}{str(v)}{Style.RESET_ALL}` and  f`{get_colot}{str(v)[0]}{Style.RESET_ALL}`